### PR TITLE
ui: add helm chart/manifests for conversational UI

### PR DESCRIPTION
**Pull request by Ruoyu-y**
## Description

Add the helm chart and manifests for conversational UI

## Issues

Related to issue #254

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ * ] New feature (non-breaking change which adds new functionality)

## Dependencies

`n/a`

## Tests

`n/a`

### CODE REVIEW

**Comment by Ruoyu-y** [comment]
@lianhao Please help with review.

**Review by lianhao — APPROVED** [review]

**Comment by lianhao** [comment]
let's wait for PR #249 to land first. It should fix the 'GMC Helm Installation test'.

**Comment by Ruoyu-y** [comment]
> let's wait for PR #249 to land first. It should fix the 'GMC Helm Installation test'.

Sure

File: helm-charts/common/conversational-ui/values.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  # tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
```
Comment (lianhao) [inline helm-charts/common/conversational-ui/values.yaml:23]: Could we add securityContext back here? but set it as an empty dict, and comment out the supposed `right` values. Also add a comment line here saying it's related to a bug in conversational UI and leave the bug link in the comment

Comment (Ruoyu-y) [reply helm-charts/common/conversational-ui/values.yaml:23]: Done

**Review by irisdingbj — APPROVED** [review]

**Pull request merged**